test/performance/benchmarks/broker-pubsub/prod.config

```diff
@@ -0,0 +1,61 @@
+# Create this benchmark with the mako tool: mako create_benchmark prod.config
+# Update this benchmark with the mako tool: mako update_benchmark prod.config
+# Learn more about the mako tool at
+# https://github.com/google/mako/blob/master/docs/CLI.md.
+
+project_name: "Knative"
+benchmark_name: "PubSub Broker Latency & Throughput"
+description: "Measure latency and throughput of the broker using various channels."
+benchmark_key: '6592390798245888'
+
+# Human owners for manual benchmark adjustments.
+owner_list: "grantrodgers@google.com"
+owner_list: "chizhg@google.com"
+owner_list: "xiyue@google.com"
+owner_list: "gracegao@google.com"
+owner_list: "nachocano@google.com"
+owner_list: "cshou@google.com"
+
+# Anyone can add their IAM robot here to publish to this benchmark.
```

Comment:
This is the dev comment but should be the prod comment:

```
# GCP Service Accounts that can publish data to Mako. Since this is a prod
# benchmark, only the CI account should be listed here.
```

...or something to that effect.